### PR TITLE
feat: Create popper component

### LINKED ISSUE #58: Move the popper into STYLE

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

Create a Popper component to use in Annotate. We use the Popper component for the toolbar submenus.

closes #58

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [x] I have read the gliff.ai Contribution Guide.
- [x] I have requested to **pull a branch** and not from main.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [ ] If appropriate, I have bumped any version numbers.

### CODE REVIEW

File: src/Popper.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <MaterialPopper
+        open={props.open}
+        anchorEl={props.anchorEl}
+        style={{ display: "flex" }}
+        placement={props.placement}
+        modifiers={[
+          {
+            name: "offset",
+            options: {
+              offset: props.offset,
```
Comment (cooper667) [inline src/Popper.tsx:29]: Should we have defaults for offset and placement? Seems like the will be the same a lot of the time.

Comment (ChrisBaidoo) [reply src/Popper.tsx:29]: Screenshot 2022-03-28 at 09 08 49

Hmmm, I think these props are required so can't have defaults?

Comment (ChrisBaidoo) [reply src/Popper.tsx:29]: unless I'm doing something wrong

Comment (cooper667) [reply src/Popper.tsx:29]: They're only required if you _make_ them required in our version of the component?

Comment (ChrisBaidoo) [reply src/Popper.tsx:29]: Okay, I've just had looked at how we did it for the Iconbuttons tooltips, looks like there might be a work around

**Review by cooper667 — DISMISSED** [review]

**Review by joshuajames-smith — CHANGES_REQUESTED** [review]
Could we add an example into STYLE?

**Comment by ChrisBaidoo** [comment]
> 

Yup, Im on it 😀

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — DISMISSED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Review by joshuajames-smith — APPROVED** [review]

**Pull request merged**